crates/matrix-sdk/CHANGELOG.md

```diff
@@ -292,6 +292,11 @@ simpler methods:
     - The `issuer` field of `UserSession` was removed.
 - `SendHandle::media_handles` was generalized into a vector
   ([#4898](https://github.com/matrix-org/matrix-rust-sdk/pull/4898))
+- [**breaking] `QueueStorage::handle_dependent_file_upload_with_thumbnail`
+  was renamed to `handle_dependent_file_or_thumbnail_upload`. Under the
+  `unstable-msc4274` feature, it was generalized to allow chaining multiple
+  dependent file / thumbnail uploads.
+  ([#4897](https://github.com/matrix-org/matrix-rust-sdk/pull/4897))
```

Comment:
This is a private API, so no need to include in the `CHANGELOG` \o/

Reply:
Oh, right! Sorry, I keep getting these changelogs wrong. 🤦‍♂️

Reply:
No worries, better to include in changelogs by default, thanks for thinking about embedders :)